<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobListWidgetItem</class>
 <widget class="QWidget" name="JobListWidgetItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>39</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="ElidedLabel" name="nameLabel">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
font-family: Monospace, Monaco;
font-size: 12px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="elide" stdset="0">
      <string notr="true">Qt::ElideRight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ElidedLabel" name="detailLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: grey;
font-family: Monospace, Monaco;
font-size: 11px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="elide" stdset="0">
      <string notr="true">Qt::ElideRight</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="ElidedLabel" name="lastBackupLabel">
     <property name="toolTip">
      <string>Last backup timestamp</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: grey;
font-family: Monospace, Monaco;
font-size: 11px;
}</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="elide" stdset="0">
      <string notr="true">Qt::ElideRight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="backupButton">
     <property name="minimumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Initiate a backup for this Job &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border: none;
border-radius: 4px;
}

QToolButton:hover {
background: rgba(179, 190, 255, 120);
}

QToolButton:pressed {
padding-top: 2px;
padding-left: 2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/tarsnap-icon.png</normaloff>:/icons/tarsnap-icon.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="inspectButton">
     <property name="minimumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Display details for this Job &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border: none;
border-radius: 4px;
}

QToolButton:hover {
background: rgba(179, 190, 255, 120);
}

QToolButton:pressed {
padding-top: 2px;
padding-left: 2px;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="restoreButton">
     <property name="minimumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Restore latest backup for this Job &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border: none;
border-radius: 4px;
}

QToolButton:hover {
background: rgba(179, 190, 255, 120);
}

QToolButton:pressed {
padding-top: 2px;
padding-left: 2px;
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/action-undo.png</normaloff>:/icons/action-undo.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="minimumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>27</width>
       <height>27</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Delete this Job &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border: none;
border-radius: 4px;
}

QToolButton:hover {
background: rgba(179, 190, 255, 120);
}

QToolButton:pressed {
padding-top: 2px;
padding-left: 2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionJobRestore">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/action-undo.png</normaloff>:/icons/action-undo.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore latest backup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionJobInspect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Inspect</string>
   </property>
   <property name="toolTip">
    <string>Show job details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionJobBackup">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/tarsnap-icon.png</normaloff>:/icons/tarsnap-icon.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Execute backup for job(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionJobDelete">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete jobs(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>../widgets/elidedlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
